.golangci.yml

```diff
@@ -0,0 +1,67 @@
+
+linters-settings:
+  cyclop:
+    max-complexity: 30
+  wsl:
+    allow-cuddle-declarations: true
+    allow-assign-and-anything: true
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - errcheck
+    - gosimple
+    - govet
+    - ineffassign
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unused
+    - varcheck
+    - dogsled
+    - gocritic
+    - gofmt
+    - goimports
+    - misspell
+    - nakedret
+    - stylecheck
+    - unconvert
+    - unparam
+    - revive
+    - ifshort
+    - whitespace
+    - wastedassign
+    - asciicheck
+    - bodyclose
+    - durationcheck
+    - errorlint
+    - exhaustive
+    - exportloopref
+    - forcetypeassert
+    - gocyclo
+    - godot
+    - goheader
+    - goprintffuncname
+    - importas
+    - makezero
+    - nestif
+    - nilerr
+    - nolintlint
+    - prealloc
+    - predeclared
+    - rowserrcheck
+    - sqlclosecheck
+    - thelper
+    - tparallel
+    - cyclop
+    - wsl
```

Comment:
sort please

Reply:
done, but why?

Reply:
thanks, large unsorted lists are an eyesore and feel like they're just a bit heavier. I know its a sucky request but it definitely helps me.